com.doycho.euterpe.gtk.json

```diff
@@ -0,0 +1,68 @@
+{
+    "app-id" : "com.doycho.euterpe.gtk",
+    "runtime" : "org.gnome.Sdk",
+    "runtime-version" : "3.38",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "euterpe-gtk",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
```

Comment:
I haven't done a build yet to check but you're very likely to want --device=dri here for gpu acceleration given the stack you're using.

Reply:
I've added it. To be honest I am pretty new to Flatpak so I am absolutely sure what I am doing. But from reading the docs I would love OpenGL rendering if that is an option. The above were what Gnome Builder generated and what else I needed to make the app do its job like access to pulseaudio.

Reply:
Yeah, device=dri will give you opengl or vulkan acceleration 'for free' with recent gtk.